privacyidea/lib/tokens/webauthntoken.py

```diff
@@ -0,0 +1,1195 @@
+        return int(get_from_config(WEBAUTHNCONFIG.CHALLENGE_VALIDITY_TIME,
+                                   get_from_config('DefaultChallengeValidityTime', 120)))
+
+    @staticmethod
+    def _get_nonce():
+        return geturandom(32)
+
+    @staticmethod
+    def get_class_type():
+        """
+        Returns the internal token type identifier
+
+        :return: webauthn
+        :rtype: basestring
+        """
+        return "webauthn"
+
+    @staticmethod
+    def get_class_prefix():
+        """
+        Return the prefix, that is used as a prefix for the serial numbers.
+
+        :return: WAN
+        :rtype: basestring
+        """
+        return "WAN"
+
+    @staticmethod
+    @log_with(log)
+    def get_class_info(key=None, ret='all'):
+        """
+        returns a subtree of the token definition
+
+        :param key: subsection identifier
+        :type key: string
+        :param ret: default return value, if nothing is found
+        :type ret: user defined
+        :return: subsection if key exists or user defined
+        :rtype: dict or scalar
+        """
+        res = {
+            'type': 'webauthn',
+            'title': 'WebAuthn Token',
+            'description': 'WebAuthn: Enroll a Web Authentication token.',
+            'init': {},
+            'config': {},
+            'user': ['enroll'],
+            # This tokentype is enrollable in the UI for...
+            'ui_enroll': ["admin", "user"],
+            'policy': {
+                SCOPE.AUTH: {
+                    WEBAUTHNACTION.ALLOWED_TRANSPORTS: {
+                        'type': 'str',
+                        'desc': _("A list of transports to prefer to communicate with WebAuthn tokens."
+                                  "Default: usb ble nfc internal lightning (All standard transports)")
+                    },
+                    WEBAUTHNACTION.TIMEOUT: {
+                        'type': 'int',
+                        'desc': _("The time in seconds the user has to confirm authorization on his WebAuthn token. " 
+                                  "Note: You will want to increase the ChallengeValidityTime along with this. "
+                                  "Default: 60")
+                    },
+                    WEBAUTHNACTION.USER_VERIFICATION_REQUIREMENT: {
+                        'type': 'str',
+                        'desc': _("Whether the user's identity should be verified when authenticating with a WebAuthn "
+                                  "token. Default: preferred (verify the user if supported by the token)"),
+                        'value': [
+                            'required',
+                            'preferred',
+                            'discouraged'
+                        ]
+                    },
+                    ACTION.CHALLENGETEXT: {
+                        'type': 'str',
+                        'desc': _('Use an alternate challenge text for telling the '
```

Comment:
Isn't it an *alternative* text? Or is it a text, that changes? 
Which text is it by default?